Assets/HoloToolkit/Input/Tests/Scripts/FocusedObjectMessageReceiver.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/// <summary>
+/// FocusedObjectMessageReceiver class shows how to handle messages sent by FocusedObjectMessageSender.
+/// This particular implementatoin controls object appearance by changing its color when focused.
+/// </summary>
+public class FocusedObjectMessageReceiver : MonoBehaviour
```

Comment:
@jwittner Yes, the value is having the possibility of multiple object selection policies and educating developers about potentially writing their own message senders and receivers, which IMHO keeps the modular spirit of HoloToolkit.